presidio-analyzer/presidio_analyzer/__main__.py

```diff
@@ -16,12 +15,20 @@
 
 sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
-from analyzer_engine import AnalyzerEngine # noqa
-from recognizer_registry.recognizer_registry import RecognizerRegistry # noqa
-from nlp_engine.spacy_nlp_engine import SpacyNlpEngine # noqa
-from presidio_logger import PresidioLogger # noqa
-
-logging.getLogger().setLevel("INFO")
+from presidio_analyzer.protobuf_models import analyze_pb2, analyze_pb2_grpc # noqa
+from presidio_analyzer import AnalyzerEngine, PresidioLogger, RecognizerRegistry # noqa
+from presidio_analyzer.nlp_engine import SpacyNlpEngine # noqa
+
+log_level_name = os.environ.get('LOG_LEVEL', 'INFO')
+log_level = logging.INFO
+if log_level_name == 'INFO':
+    log_level = logging.INFO
+if log_level_name == 'WARNING':
+    log_level = logging.WARNING
+if log_level_name == 'ERROR':
+    log_level = logging.ERROR
```

Comment:
I think you can do something like that:
```
log_level_name = os.environ.get('LOG_LEVEL', 'INFO').upper()
logging.basicConfig(level=log_level_name)
```
WDYT?

Reply:
it's just to verify that it doesn't break if the user puts something illegal on LOG_LEVEL, but if you think it makes more sense I have no problem with changing it to your suggestion.

Reply:
Ok, so I think we can delete then lines 24+25

Reply:
Done